cli/package.json

```diff
@@ -7,7 +7,7 @@
     "cypress": "bin/cypress"
   },
   "engines": {
-    "node": ">=4.0.0"
+    "node": "11.6.0"
```

Comment:
this is a problem - Renovate should not pin or change the node engines we specify

Reply:
trying to confirm that Renovate tries to upgrade `engines` field in `package.json` in this example repo https://github.com/bahmutov/test-renovate-monorepo/commit/01b5445e25d0b23216b088a46a4d9a03bd571065 and that using option `node: {enabled: false}` would disable it according to this doc https://renovatebot.com/docs/node/